marko.json

```diff
@@ -89,6 +89,29 @@
         "@style": "string",
         "@disabled": "boolean"
     },
+    "<ebay-combobox>": {
+        "renderer": "./src/components/ebay-combobox/index.js",
+        "@*": "expression",
+        "@html-attributes": "expression",
+        "@class": "string",
+        "@style": "string",
+        "@name": "string",
+        "@grow": "boolean",
+        "@borderless": "boolean",
+        "@disabled": "boolean",
+        "@options <option>[]": {
+            "@*": "expression",
+            "@html-attributes": "expression",
+            "@class": "string",
+            "@style": "string",
+            "@value": "string",
+            "@text": "string",
+            "@selected": "boolean"
```

Comment:
Let's use `autofill` instead of selected

Reply:
Actually, see this comment: https://github.com/eBay/ebayui-core/pull/566#issuecomment-482728718